torch/_inductor/fx_passes/quantization.py

```diff
@@ -2758,13 +2759,19 @@ def qconv(match: Match, *args, **kwargs):
             else:
                 accum = (
                     kwargs["accum"]
-                    if output_dtype == torch.uint8
+                    if output_dtype in OrderedSet([torch.uint8, torch.int8])
```

Comment:
`OrderedSet` has been removed from this file at the last main branch.
I think using a list is enough.
```suggestion
                    if output_dtype in [torch.uint8, torch.int8]
```

Reply:
thanks for reminding, has modified